api/reference/network_routers.adoc

```diff
@@ -0,0 +1,71 @@
+
+[[network-routers]]
+== Network Routers
+
+Management of Network Routers is provided via the following collection:
+
+[source,data]
+----
+/api/network_routers
+----
+
+The following actions are available on network_routers:
+
+* link:#querying-network-routers[Querying Network Routers]
+* link:#deleting-network-routers[Deleting Network Routers]
+
+[[querying-network-routers]]
+=== Querying Network Routers
+
+Querying all network routers in the system is simply:
+
+----
+GET /api/network_routers
+----
+
+Getting details on a specific Network Router:
+
+----
+GET /api/network_routers/:id
+----
+
+[[deleting-network-routers]]
+=== Deleting Network Routers
+
+Network Routers can be deleted via either the *delete* POST action or via the DELETE HTTP method.
+
+----
+POST /api/network_routers/101
```

Comment:
Not sure if this is consistent across the board, but in most places where a task result is returned, the docs show a response example. Should we have one here?